src/partials/editor.html

```diff
@@ -1,5 +1,17 @@
 <div class="editor-row">
   <div class="section gf-form-group">
+    <div class="grafana-info-box">
+      <h5>Migration</h5>
+      <p>
+        Consider switching to the <a href="https://grafana.com/docs/grafana/latest/panels-visualizations/visualizations/geomap/" target="_blank">Geomap</a> visualization type. It is a more capable and performant version of this panel.
+      </p>
+      <p>
+        <button class="btn btn-primary" ng-click="ctrl.migrateToReact()">
+          Migrate
+        </button>
+      </p>
+    </div>
```

Comment:
Should include a disclaimer like in graph panel that full feature parity may not exist for the time being (i.e. missing some icons, tooltip experience UX, etc.)? Similar to what we do for Graph panel. in addition / alternatively, this could be a place we consider linking to the [geomap github discussion](https://github.com/grafana/grafana/discussions/62159) to collect direct feedback on missing parity items from worldmap to geomap 🤔 (not sure on exact messaging here, but maybe something along the lines of "something missing from Geomap that you rely on in worldmap? let us know here"

<img width="528" alt="Screenshot 2023-06-08 at 8 22 58 PM" src="https://github.com/grafana/worldmap-panel/assets/22381771/6650f5d2-892c-49d6-a328-8f9f3effcc2c">

Reply:
Could probably "advertise" some of the major new features.

Reply:
Agree maybe we can add a sentence enticing people to switch to Geomap at the end of the of the first paragraph and then have the messaging around if something is missing to let us know in the discussion (in the section after the button)? I'm good to defer to whatever you think is best - approving PR :)

Reply:
I'm happy with this. 
![image](https://github.com/grafana/worldmap-panel/assets/60050885/4e01cb0a-89b6-42b9-ab40-b9b8c677555e)